baselines/aad.md

```diff
@@ -23,21 +23,20 @@ or [G3](https://www.microsoft.com/en-us/microsoft-365/government)
 license level. Therefore, only licenses not included in E3/G3 are
 listed.
 
-Some of the policies in this baseline may link to Microsoft instruction pages which assume that an agency has created emergency access accounts in AAD and [implemented strong security measures](https://docs.microsoft.com/en-us/azure/active-directory/roles/security-emergency-access) to protect the credentials of those accounts.
+Some of the policies in this baseline may link to Microsoft instruction pages which assume that an agency has created emergency access accounts in AAD and [implemented strong security measures](https://learn.microsoft.com/en-us/azure/active-directory/roles/security-emergency-access) to protect the credentials of those accounts.
 
 ## Key Terminology
 
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
 The following are key terms and descriptions used in this document.
 
 **Hybrid Azure Active Directory (AD)** – This term denotes the scenario
 when an organization has an on-premises AD domain that contains the
 master user directory but federates access to the cloud Microsoft 365
 (M365) Azure AD tenant.
 
-**Resource Tenant & Home Tenant** – In scenarios where [guest users are involved](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/authentication-conditional-access) the resource tenant hosts the M365 target resources that the guest user is accessing. The home tenant is the one that hosts the guest user's identity.
-
-**Home Tenant** – In scenarios where guest users are involved, the
-[home tenant](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/authentication-conditional-access) is the one that hosts the guest user’s identity.
+**Resource Tenant & Home Tenant** – In scenarios where [guest users are involved](https://learn.microsoft.com/en-us/azure/active-directory/external-identities/authentication-conditional-access) the **resource tenant** hosts the M365 target resources that the guest user is accessing. The **home tenant** is the one that hosts the guest user's identity.
 
 ## Highly Privileged Roles
```

Comment:
(nit)

Please list each highly privileged role on a new line.  The current format is strange IMHO.

Reply:
Fixed.